src/cmds/net/telnetd/telnetd.c

```diff
@@ -169,16 +168,14 @@ static void *shell_hnd(void* args) {
 		MD(printf("dup2 STDERR_FILENO error: %d\n", errno));
 	}
 
-	ret = shell_run(shell_lookup("tish"));
-	if (ret != 0) {
-		MD(printf("shell_run error: %d\n", ret));
-	}
+	system("tish");
```

Comment:
@AntonKozlov  It will be fix with pthread using in a main service after implementation correct FD_CLOEXEC treating